returnn/forward.py

```diff
@@ -47,7 +47,8 @@ def __init__(
     ):
         """
 
-        :param model_checkpoint: Checkpoint object pointing to a stored RETURNN Tensorflow model
+        :param model_checkpoint: Checkpoint object pointing to a stored RETURNN Tensorflow model or None if network has
+          no parameters
```

Comment:
`... or should be randomly initialized`

I found out this week there are actually use cases for that... (e.g. profiling)